fair_mango/metrics/metrics.py

```diff
@@ -295,6 +890,72 @@ def __init__(
 
 
 class EqualOpportunityDifference(FairnessMetricDifference):
+    """Calculate Equal Opportunity Fairness Metric using "difference" to
+    calculate the disparity between the different demographic groups present
+    in the sensitive feature.
+
+    Equal Opportunity calculates the "difference" in the True Positive Rate in
+    the targets to detect if there is any bias in the **model**.
```

Comment:
maybe remind what question is the True Positive Rate answering so we can quickly understand this metric better

Reply:
I added more detailed description to every fairness metric explaining the score it compares.